test/media/drm_engine_unit.js

```diff
@@ -2164,6 +2164,26 @@ function testDrmEngine(useMediaCapabilities) {
     });
   });
 
+  describe('isFairPlayKeySystem', () => {
+    it('should return true for FairPlay', () => {
+      expect(shaka.media.DrmEngine.isFairPlayKeySystem(
+          'com.apple.fps')).toBe(true);
+      expect(shaka.media.DrmEngine.isFairPlayKeySystem(
+          'com.apple.fps.1_0g')).toBe(true);
```

Comment:
Is that meant to have a "g" on the end of it?

Reply:
It's a typo sorry :(